ui/src/app/tasks-jobs/tasks/destroy/destroy.component.html

```diff
@@ -1,24 +1,16 @@
 <clr-modal [(clrModalOpen)]="isOpen" *ngIf="tasks" [clrModalClosable]="!isRunning">
-  <h3 *ngIf="tasks?.length === 1" class="modal-title">Confirm Destroy Task</h3>
-  <h3 *ngIf="tasks?.length > 1" class="modal-title">Confirm Destroy Tasks</h3>
+  <h3 *ngIf="tasks?.length === 1" class="modal-title">{{ 'tasks.destroy.destroyTitle' | translate }}</h3>
+  <h3 *ngIf="tasks?.length > 1" class="modal-title">{{ 'tasks.destroy.destroyTitle2' | translate }}</h3>
   <div class="modal-body" *ngIf="tasks.length === 1 && !isRunning">
-    This action will destroy and delete the task
-    <u
-      ><strong>{{ tasks[0].name }}</strong></u
-    >. Are you sure?
+    <div [innerHTML]="'tasks.destroy.content' | translate: {name: tasks[0].name}"></div>
   </div>
   <div class="modal-body" *ngIf="tasks?.length > 1 && !isRunning">
-    This action will destroy the
-    <u
-      ><strong>{{ tasks.length }} task{{ tasks?.length > 1 ? 's' : '' }}</strong></u
-    >
-    listed below.<br />Are you sure?
-    <br />
+    <div [innerHTML]="'tasks.destroy.content2' | translate: {count: tasks?.length}"></div>
     <table class="table table-striped">
       <thead>
         <tr>
-          <th class="left">Name</th>
-          <th class="left">Type</th>
+          <th class="left">{{ 'tasks.main.name' | translate }}</th>
```

Comment:
Texts like `Name` and `Type` are visible on other pages.
Can we move these to `commons` translations ?